app.ts

```diff
@@ -1,16 +1,16 @@
 //@ts-ignore
-import tunnelmoleConnections from './src/handlers/tunnelmole-connections';
-import handleRequest from './src/handlers/handle-request';
-import logTelemetry from './src/handlers/log-telemetry';
+import tunnelmoleConnections from "./src/handlers/tunnelmole-connections";
```

Comment:
Seems to be formatting changes only, are these needed?

Reply:
Nope, just [prettier](https://prettier.io/)